auditbeat/docs/running-on-docker.asciidoc

```diff
@@ -0,0 +1,20 @@
+include::../../libbeat/docs/shared-docker.asciidoc[]
+
+[float]
+==== Special requirements
+
+Under Docker, {beatname_uc} runs as a non-root user, but requires some privileged
+capabilities to operate correctly. Ensure that the +AUDIT_CONTROL+ and +AUDIT_READ+
+capabilities are available to the container.
+
+["source","sh",subs="attributes"]
+----
+docker run --cap-add=AUDIT_CONTROL,AUDIT_READ {dockerimage}
+----
+
+It is also essential to run {beatname_uc} in the host PID namespace.
+
+["source","sh",subs="attributes"]
+----
+docker run --pid=host {dockerimage}
```

Comment:
I expect people will copy/paste this so I recommend showing both the `-cap-add` and `--pid=host` on the same line.

Reply:
Good point, I was trying to be minimal to draw attention to the current topic, but you're probably right about cut-n-paste.

Reply:
Fixed in 29f4c6fe3ee